benchmarks/apfel_bench.py

```diff
@@ -5,7 +5,7 @@
 import numpy as np
 from banana.data import cartesian_product
 
-from ekomark.banana_cfg import register
+from ekomark import register
```

Comment:
I think  `register` moved into banana: https://github.com/N3PDF/banana/commit/b51feaa2e2d31f41b73ec30f421fc36e08d79f9b

Reply:
I see, but then we need to remove this: 
https://github.com/N3PDF/eko/blob/6accf8b20333d3b56a0b12c40a5e181191be58cb/src/ekomark/__init__.py#L13

Reply:
Yes, I initially fixed for `ekomark` only, the moment I had to do it for `yadmark` I did the proper thing in `banana`, but `ekomark` has still to be updated.